pySDC/implementations/problem_classes/AllenCahn_1D_FD.py

```diff
@@ -663,21 +672,24 @@ def eval_f(self, u, t):
         self.work_counters['rhs']()
         return f
 
-    def u_exact(self, t):
+    def u_exact(self, t, u_init=None, t_init=None):
```

Comment:
Actually, I don't think this function approximates anything with scipy... Please either remove the added comments and replace `u_init` and `t_init` with `**kwargs` or implement a call to the function generating scipy reference solutions in the core problem class.

Reply:
Docstring has been corrected

Reply:
Should have been more precise about the role of `u_init` and `t_init` earlier. Sorry for that!